### PR TITLE
Set up and verify validator with validator-keys tool

### PR DESCRIPTION
**Pull request by wilsonianb**
Domain verification instruction depend on setting up validators@ripple.com email address and rippled release that include `validator-keys` tool `sign` function.

### CODE REVIEW

File: content/tutorial-rippled-setup.md [hunk 1]
```diff
@@ -202,33 +202,28 @@ Running a `rippled` validator that participates in the Consensus process is simp
 
 ## Validator Setup ##
 
-1. [Install a `rippled` server.](#installing-rippled)
+The `validator-keys` tool (included in the rippled rpm) is the recommended means to securely generate and manage your validator keys.
```
Comment (mDuo13) [inline content/tutorial-rippled-setup.md]: Put `rippled` in code font and RPM in caps

File: content/tutorial-rippled-setup.md [hunk 2]
```diff
@@ -202,33 +202,28 @@ Running a `rippled` validator that participates in the Consensus process is simp
...
 
-4. Edit your `rippled.cfg` file to add the `validation_seed` value you generated in step 3:
+3. Generate a validator token and edit your `rippled.cfg` file to add the `[validator_token]` value.
 
-        [validation_seed]
-        ssdecohJMDPFuUPDkmG1w4objZyp4
+        $ /opt/ripple/bin/validator-keys create_token --keyfile /path/to/your/validator-keys.json
 
-    **Warning:** Be sure to use your own unique random seed, and keep it secure! If others know your `validation_seed`, they can forge validations from your server.
+    If you had previously configured your validator without using the `validator-keys` tool, you will need to also delete the `[validation_seed]` from your `rippled.cfg` file. This will change your validator public key.
```
Comment (mDuo13) [inline content/tutorial-rippled-setup.md]: > will change

Use "changes" (present tense) instead.

File: content/tutorial-rippled-setup.md [hunk 3]
```diff
@@ -253,46 +248,22 @@ Take care not to publish the IP address of your validator.
...
+        $ /opt/ripple/bin/rippled server_info -q | grep pubkey_validator
 
-A master seed can be used to generate both a validation public key and a Ripple account address. Since the same secret key is used for both, whoever operates the validator also controls the account with the corresponding address. (The validator's public key and the account address both represent the public key for the same keypair.)
+2. Sign the validator public key (from step 1) using the SSL private key used for your domain. The SSL private key file does not need to be stored on the validator server.
 
-The steps below describe how to set the domain field of a validator's Ripple account.
+        $ openssl dgst -sha256 -hex -sign /path/to/your/ssl.key <(echo <your-validator-public-key>)
 
-1. Get the validator's account address (`account_id`) using the `validation_seed` generated [above](#validator-setup) in step 3:
+3. Using `validator-keys` tool (included in the rippled rpm), sign the domain name:
```
Comment (mDuo13) [inline content/tutorial-rippled-setup.md]: > (included in the rippled rpm)

put `rippled` in code font and RPM in caps here too please

File: content/tutorial-rippled-setup.md [hunk 4]
```diff
@@ -202,33 +202,28 @@ Running a `rippled` validator that participates in the Consensus process is simp
...
 
-        [validation_seed]
-        ssdecohJMDPFuUPDkmG1w4objZyp4
+        $ /opt/ripple/bin/validator-keys create_token --keyfile /path/to/your/validator-keys.json
 
-    **Warning:** Be sure to use your own unique random seed, and keep it secure! If others know your `validation_seed`, they can forge validations from your server.
+    If you had previously configured your validator without using the `validator-keys` tool, you will need to also delete the `[validation_seed]` from your `rippled.cfg` file. This will change your validator public key.
 
-5. Restart `rippled` validator:
+4. Start `rippled` validator:
```
Comment (mDuo13) [inline content/tutorial-rippled-setup.md]: I think it makes sense to drop the word "validator" from this line

**Review by mDuo13 — APPROVED** [review]
Left a couple minor suggestions for wording and formatting. Assuming those are fixed (and the other prerequisites are ready), I'm OK with merging this.

**Comment by wilsonianb** [comment]
Updated with fixes

**Comment by warpaul** [comment]
Thanks @mDuo13. Can we get this merged and up on dev portal asap? It's a key enabler for domain verification.

**Review by mDuo13 — APPROVED** [review]
Updated version looks good.

**Pull request merged**